src/Compilers/CSharp/Portable/CSharpResources.Designer.cs

```diff
@@ -8567,6 +8567,15 @@ internal class CSharpResources {
         }
         
         /// <summary>
+        ///   Looks up a localized string similar to The source does not appear to be valid C# code..
+        /// </summary>
+        internal static string ERR_WrongLanguage {
```

Comment:
:exclamation: Seems like this change was not intentional. Should it be reverted prior to merge?